erdpy/validators/validators.py

```diff
@@ -1,13 +1,21 @@
 import binascii
 import logging
 
+from erdpy import config
 from erdpy.accounts import Address
 
 logger = logging.getLogger("validators")
 
 _STAKE_SMART_CONTRACT_ADDRESS = "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l"
 
 
+def estimate_gas_limit(args, num_keys):
```

Comment:
Rename to `estimate_system_sc_call(data, base_cost, factor)` perhaps (optional, of course)? And it can be called as follows:

```
estimate_system_sc_call(args.data, config.SystemSC_Stake_Cost or whatever, len(num_nodes) or num_keys or whatever)
```

Reply:
done.